lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -281,11 +307,29 @@ def _configure_nodes(self, environment: Environment, log: Logger) -> None:
                 vm_disks_dir, f"{node_context.vm_name}-cloud-init.iso"
             )
             node_context.os_disk_base_file_path = qemu_node_runbook.qcow2
+            if self.qemu_platform_runbook.is_host_remote():
+                host = self.qemu_platform_runbook.host
+                new_os_disk_base_file_path = os.path.join(
+                    "/home",
+                    host.username,
+                    vm_name_prefix,
+                    os.path.basename(qemu_node_runbook.qcow2)
+                )
+
+                local_node.execute(
```

Comment:
There shouldn't be anything in  `_configure_nodes` that will require cleanup, such as creating files. Just make a note of where everything will be placed and then do the actual work in `_create_nodes`.

Reply:
For example, you could create a new node variable called `os_disk_source_file_path` and only set that variable if the file needs to be copied to the host. Then `_create_nodes` can copy the file if that variable is not `None`.

Reply:
Done